src/main/java/io/stargate/sgv2/jsonapi/service/processor/CommandProcessor.java

```diff
@@ -46,36 +60,92 @@ public CommandProcessor(
    */
   public <T extends Command> Uni<CommandResult> processCommand(
       CommandContext commandContext, T command) {
-    // start by resolving the command, get resolver
-    return commandResolverService
-        .resolverForCommand(command)
-
-        // resolver can be null, not handled in CommandResolverService for now
-        .flatMap(
-            resolver -> {
-              // if we have resolver, resolve operation and execute
-              Operation operation = resolver.resolveCommand(commandContext, command);
-              return operation.execute(queryExecutor);
-            })
+    final Tag commandTag = Tag.of(MetricsConstants.COMMAND, command.getClass().getSimpleName());
+    final String tenant =
+        stargateRequestInfo != null && stargateRequestInfo.getTenantId().isPresent()
+            ? stargateRequestInfo.getTenantId().get()
+            : MetricsConstants.UNKNOWN_VALUE;
```

Comment:
`stargateRequestInfo` is never null here.. just `stargateRequestInfo.getTenantId().orElse(MetricsConstants.UNKNOWN_VALUE)`